ce/generators/msbuild-generator.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+const XMLWriter = require('xml-writer');
```

Comment:
There aren't shiny TypeScript bindings for this so it gets red squiggles but the API is simple enough this seems OK? Or if there's a better way to write it as an import rather than a require I'd rather do that.